src/SourceGenerators/Uno.UI.SourceGenerators.Internal/DependencyObject/DependencyPropertyGenerator.cs

```diff
@@ -121,7 +121,7 @@ private record GenerationCandidateData
 
 			public GenerationCandidateData(ISymbol dpSymbol, AttributeData attribute)
 			{
-				PropertyName = dpSymbol.Name.TrimEnd("Property", StringComparison.Ordinal);
```

Comment:
I removed the `TrimEnd` extension that accepts `StringComparison` since the other overload already uses `StringComparison.Ordinal`.